src/components/Form.js

```diff
@@ -1,12 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
 function Form() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const dispatch = useDispatch();
+  const addEachBook = (event) => {
+    event.preventDefault();
+    dispatch(addBook({ id: uuidv4(), title, author }));
+  };
   return (
     <>
       <h4>Add A form</h4>
-      <form>
-        <input type="text" name="" id="" placeholder="Book Title" />
-        <input type="text" name="" id="" placeholder="Author" />
+      <form onSubmit={addEachBook}>
+        <input type="text" name="" id="" placeholder="Book Title" onChange={(e) => setTitle(e.target.value)} />
+        <input type="text" name="" id="" placeholder="Author" onChange={(e) => setAuthor(e.target.value)} />
```

Comment:
- [ ] (OPTIONAL) It would be great if you could validate your inputs so that a user would not be able to submit a book without a title or an author 👍